src/com/rakovets/course/practice/module3/Task01.java

```diff
@@ -23,5 +23,10 @@ public static void main(String[] args) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используяся переменные объявленные выше (их можно изменять)
         // Для проверки решения необходимо запустить @Test для данного class (в директории test)
+
+        if (positionPlayerByDamage < 6 && positionPlayerByDamage >= 1)

```

Comment:
INFO: violation of code convention
Please, use:
```suggestion
if () {
    //something
} else {
   //something
}
```